buildozer/runner.bash.template

```diff
@@ -4,7 +4,15 @@ set -euo pipefail
 
 BUILDOZER_SHORT_PATH=@@BUILDOZER_SHORT_PATH@@
 ARGS=@@ARGS@@
+ERROR_ON_NO_CHANGES=@@ERROR_ON_NO_CHANGES@@
 
 buildozer_short_path=$(readlink "$BUILDOZER_SHORT_PATH")
 cd "$BUILD_WORKSPACE_DIRECTORY"
+set +e
 "$buildozer_short_path" "${ARGS[@]}"
+ret=$?
+if [[ "$ret" -eq 3 && "$ERROR_ON_NO_CHANGES" == "false" ]]; then
```

Comment:
heh, but it's just so... bash-y

(I like using test - also, even if I was using bash, I'd prefer '=' over '==', since that would be valid in test as well)

Reply:
Obviously I'm not closely familiar with other shells and ways to make script more "cross-shell" =) If this does not have (obvious) bugs, I'll merge it.

Reply:
Thanks for explaining. I suspected something like that.
If you want to refactor any/all scripts in this repo - feel free to send PRs. I'm not opposed to changing these scripts in any way that makes sense to you.